Fighter.cpp

```diff
@@ -5,13 +5,34 @@ void Fighter::take_dmg(Fighter& enemy) {
 	HP -= enemy.getDMG();
 	if (HP < 0)
 		HP = 0;
+	enemy.exp += enemy.getDMG();
+
+	/*level trigger*/
+	if (enemy.exp>=100)
+	{
+		
+		int u = int(enemy.exp / 100);
+		enemy.level+=u;
+		enemy.exp = enemy.exp%100;
+		
+		levelUP(&enemy);

```

Comment:
A level annyival nő, amennyivel kell, de a levelUP nem hívódik meg többször, ha több szintet is lép egyszerre